Studio-Lovelies/GG-JointJustice-Unity
# fix(472): Add padding to the dialogue box

## Summary

This PR fixes bug #472 by adding padding to the dialogue box. This is a Unity-only fix, no code, as I just changed the dimensions of the dialogue box.

- **Cons**: The padding will now only allow 3 lines of dialogue instead of 4.
- **Pros**: The dialogue will never overlap with the ContinueArrow.

In my opinion, 4 lines of dialogue looks kind of crowded anyways. 3 lines with more padding makes it look a little less crowded and "textbook"-y.

## Before/after screenshots and/or animated gif

### Before:
![image](https://github.com/user-attachments/assets/322e99d1-3109-4da4-becf-c89841a37687)
<img width="388" alt="image" src="https://github.com/user-attachments/assets/751919de-181f-4a3e-818f-0732d169e293">

### After:
![paddedDialogue](https://github.com/user-attachments/assets/9a18d0be-59d1-4009-ab80-0093a084d5ff)
<img width="681" alt="image" src="https://github.com/user-attachments/assets/9b7bcaaf-d0b8-4a51-b1d0-9596f59ee224">
<img width="385" alt="image" src="https://github.com/user-attachments/assets/5aa0b7e8-31ba-4663-84aa-3157f5848833">

## Testing instructions

- Run any part of the game and observe the textbox
- Check to see if anything looks awkward or misaligned
- Also observe that the text will never overlap with the ContinueButton

## Additional information

- [ ] Introduces new feature
- [ ] Removes existing feature
- [x] Has associated resource: https://github.com/Studio-Lovelies/GG-JointJustice-Unity/issues/472
